linera-sdk/src/contract/runtime.rs

```diff
@@ -201,6 +201,34 @@ where
         bcs::from_bytes(&response_bytes)
             .expect("Failed to deserialize `Response` type from cross-application call")
     }
+
+    /// Queries our application service as an oracle and returns the response.
+    ///
+    /// Should only be used with queries where it is very likely that all validators will compute
+    /// the same result, otherwise most block proposals will fail.
+    ///
+    /// Cannot be used in fast blocks: A block using this call should be proposed by a regular
+    /// owner, not a super owner.
```

Comment:
I believe super owners can propose slow blocks, though :)

Reply:
Yes, but our client software will automatically propose a fast block if it can. Or rather, it will just fail to do anything, because it will fail to execute the fast block locally.